core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -553,6 +554,35 @@ impl ConfigurableDatabase for RocksDBStore {
     }
 }
 
+impl MeasurableDatabase for RocksDBStore {
+    fn get_data_volume_statistics(&self) -> Vec<CategoryDbVolumeStatistic> {
+        let mut statistics = ALL_COLUMN_FAMILIES
+            .iter()
+            .map(|cf_name| {
+                (
+                    cf_name.to_string(),
+                    CategoryDbVolumeStatistic::zero(cf_name.to_string()),
+                )
+            })
+            .collect::<IndexMap<_, _>>();
+        let live_files = match self.db.live_files() {
+            Ok(live_files) => live_files,
+            Err(err) => {
+                warn!("could not get DB live files; returning 0: {:?}", err);
+                Vec::new()
+            }
+        };
+        for live_file in live_files {
+            let Some(statistic) = statistics.get_mut(&live_file.column_family_name) else {
+                warn!("LiveFile of unknown column family: {:?}", live_file);
+                continue;
+            };
+            statistic.add(live_file.num_entries, live_file.size);
```

Comment:
As we're iterating across SSTs, I guess these entries could be tombstones as well as "create/update" entries? Which is fine, just wanted to understand what we're measuring.

It would be nice to update the description on `raw_db_entries` to include if it includes tombstones, updates, and recreations which haven't yet been compacted?

Reply:
The rocks' doc says `Number of entries/alive keys in the file`.

And I believe it is the actual number of alive keys (i.e. excludes deletions), because there is another field available:
```
    /// Number of deletions/tomb key(s) in the file
    pub num_deletions: u64,
```
(which I choose to not monitor)

Reply:
OFC the total that I am `.add()`ing is still an estimate, because of upserts. Lemme add some doc about it.

Reply:
Oooo interesting :D

I could see it be cool to return all three:
* `num_tombstones`
* `num_upserts`
* `num_entries ~ num_upserts - num_tombstones`

But anyway, not needed for 1.0.4